src/Components/Facility/Consultations/DailyRoundsFilter.tsx

```diff
@@ -0,0 +1,130 @@
+import { Popover, Transition } from "@headlessui/react";
+import ButtonV2 from "../../Common/components/ButtonV2";
+import { Fragment } from "react";
+import { SelectFormField } from "../../Form/FormFields/SelectFormField";
+import TextFormField from "../../Form/FormFields/TextFormField";
+import CareIcon from "../../../CAREUI/icons/CareIcon";
+import dayjs from "dayjs";
+import { useState } from "react";
+import { useTranslation } from "react-i18next";
+import { DailyRoundsFilterModel } from "../models";
+
+export const DailyRoundsFilter = (props: {
+  onApply: (filter: any) => void;
```

Comment:
Avoid usage of `any` when you know the type for sure.
```suggestion
  onApply: (filter: DailyRoundsFilterModel) => void;
```